### PR TITLE
Update example code to use React Class syntax

### PR DESCRIPTION
**Pull request by rubencodes**
This is the recommended syntax to use now, and as such will be more familiar to beginners. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,28 +21,24 @@ You’ll need to install React and Moment separately since they are not included
 
 ```js
 var React = require('react');
```
Comment (shlomizadok) [inline README.md]: maybe also change to `import React from 'react';` ?

**Comment by rubencodes** [comment]
Done! Was just holding off on doing so since some are still attached to the old `require` format, but I agree it makes more sense as `import`s if we're doing `class` syntax.

**Comment by shlomizadok** [comment]
LGTM!

**Comment by jasonsalzman** [comment]
Updated PR title to reflect the changes being made, since the actual package already utilitizes proper es6 syntax.

Thanks for the contribution. I'll merge this later this evening or tomorrow.

**Comment by jasonsalzman** [comment]
released with v0.1.4

**Pull request merged**